mmseg/models/decode_heads/uper_head.py

```diff
@@ -84,9 +84,12 @@ def psp_forward(self, inputs):
 
         return output
 
-    def forward_feature(self, inputs):
-        """Feature map before `self.cls_seg` and learnable semantic kernels can
-        be both output for kernel updation."""
+    def _forward_feature(self, inputs):
```

Comment:
Is change forward_feature to _forward_feature necessary? It might cause BC-breaking

Reply:
We have fixed it by: https://github.com/open-mmlab/mmsegmentation/pull/1299.